### ISSUE TITLE
Configure custom command when using compare changes

### ISSUE THREAD

**Issue by calleo** [dbt-labs/terraform-provider-dbtcloud#318]
## Is your feature request related to a problem? Please describe.

Using Compare Changes, without a custom command, always (almost) causes the comparison to time out, since dbt cloud is trying to compare massive models, row by row.

## Describe the solution you'd like

In dbt cloud, I am able to manually set a custom command to limit the amount of models. I want to set this using Terraform.

https://docs.getdbt.com/docs/deploy/job-commands#compare-changes-custom-commands

## Describe alternatives you've considered

Setting it manually in dbt cloud.

## Additional context

-

**Comment by calleo** [comment]
I suspect this is available already in the rest api, using the parameter `compare_changes_flags`.

**Comment by b-per** [comment]
Yes, it is in the process of being rolled out for all customers so we should be able to add it now.

**Comment by b-per** [comment]
Released in 0.3.24
